_sass/minima/dark-mode.scss

```
@@ -82,3 +82,18 @@ table td{
 canvas {
     filter: invert(100%);
 }
+
+.post-list > li > div {
+    border-radius: 5px !important;
+    box-shadow: 0 3px 6px rgba(149,157,165,0.15) !important;
+    background-color: #212121;
+    border: none;
+}
```

No need to modify border properties if its being set to `none` right? :D

`background-color` can simply inherit the parent element color. Not required to hard code it.

Also, shadow on box element seems inconsistent because there is no concept of depth. Its just a minimal list of post titles.

```suggestion
.post-list > li > div {
    box-shadow: none !important;
    background-color: inherit !important;
    border: none !important;
}
```